schema.h

```diff
@@ -35,11 +35,26 @@ SOFTWARE.
 // Data structures for internal storage.
 enum class UID : uint64_t { INVALID = 0u };
 enum class CID : uint64_t { INVALID = 0u };
-enum class ANSWER : int { UNSEEN = 0, CTFO = 1, TFU = 2, TIFB = -1 };
+enum class ANSWER : int { UNSEEN = 0, CTFO = 1, TFU = 2, SKIP = -1 };
+enum class AUTH_TYPE : int { UNDEFINED = 0, IOS };
+
+const std::vector<unsigned int> LEVEL_SCORES{
```

Comment:
OMFG, it really does! Starts failing it only if `LEVEL_SCORES` is made non-const. Even passes if I take a pointer to the `const` one.

Clean test: `ln -sf schema.h 1.cc && ln -sf schema.h 2.cc && clang++ -std=c++11 -c 1.cc && clang++ -std=c++11 -c 2.cc && clang++ -o 12 1.o 2.o`

Looks like this StackOverflow answer is right: `"const" implies internal linkage`.
(http://stackoverflow.com/questions/2268749/defining-global-constant-in-c)

Thanks!
Dima